src/test/java/com/teragrep/pth_03/tests/EvalSyntaxTests.java

```diff
@@ -431,4 +432,34 @@ void xpathTest25(String arg) throws Exception {
         // Check that 1 found
         assertEquals(1, nodesA.getLength());
     }
+
+    @ParameterizedTest
+    @ValueSource(strings = {
+            "eval_sum"
+    })
+    void sumTest(String arg) throws Exception {
```

Comment:
Don't throw exceptions in tests, use assertDoesNotThrow instead

Reply:
Somehow I forgot this again. Fixed it now.